fixtures/cameo/zenit-z120.json

```diff
@@ -0,0 +1,270 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Zenit Z120",
+  "shortName": "Z120",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["EP"],
+    "createDate": "2020-06-11",
+    "lastModifyDate": "2020-06-11"
+  },
+  "links": {
+    "productPage": [
+      "https://www.cameolight.com/fr/solutions/integrateurs/eclairage-statique/projecteurs-a-par/2416/zenit-z120?c=2038#detail-downloads",
+      "https://www.cameolight.com/fr/solutions/integrateurs/eclairage-statique/projecteurs-a-par/2416/zenit-z120?c=2038#detail-downloads",
+      "https://www.cameolight.com/fr/solutions/integrateurs/eclairage-statique/projecteurs-a-par/2416/zenit-z120?c=2038#detail-downloads"
+    ]
+  },
+  "availableChannels": {
+    "Dimmer": {
+      "highlightValue": "100%",
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Strobe": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 5],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        },
+        {
+          "dmxRange": [6, 10],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Closed"
+        },
+        {
+          "dmxRange": [11, 33],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Pulse",
+          "speedStart": "slow",
+          "speedEnd": "fast",
+          "randomTiming": true
+        },
+        {
+          "dmxRange": [34, 56],
+          "type": "ShutterStrobe",
+          "shutterEffect": "RampUp",
+          "speedStart": "slow",
+          "speedEnd": "fast",
+          "randomTiming": true
+        },
+        {
+          "dmxRange": [57, 79],
+          "type": "ShutterStrobe",
+          "shutterEffect": "RampDown",
+          "speedStart": "slow",
+          "speedEnd": "fast",
+          "randomTiming": true
+        },
+        {
+          "dmxRange": [80, 102],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Strobe",
+          "speedStart": "slow",
+          "speedEnd": "fast",
+          "randomTiming": true
+        },
+        {
+          "dmxRange": [103, 127],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Strobe",
+          "speedStart": "slow",
+          "speedEnd": "fast"
+        },
+        {
+          "dmxRange": [128, 250],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Strobe",
+          "speedStart": "slow",
+          "speedEnd": "fast"
```

Comment:
"<1Hz - 20Hz" -- we could probably put absolute frequencies here.

Reply:
Looking at other fixtures, the lowest value seems to be 1Hz so I'll use that.